pa11ycrawler/pipelines.py

```diff
@@ -182,6 +221,7 @@ def process_item(self, item, spider):
                 )
             )
 
+        self.check_title_match(item['page_title'], stdout, spider.logger)
```

Comment:
No, it will log an error using `spider.logger`, but it won't raise an exception. I figured that it's better to have _some_ output than none at all, but I might be wrong on that. What do you think?